components/image-builder/leeway.Dockerfile

```diff
@@ -1,13 +1,10 @@
-# Copyright (c) 2020 Gitpod GmbH. All rights reserved.
+# Copyright (c) 2021 Gitpod GmbH. All rights reserved.
 # Licensed under the GNU Affero General Public License (AGPL).
 # See License-AGPL.txt in the project root for license information.
 
-FROM alpine:3.13
-
-# Ensure latest packages are present, like security updates.
-RUN  apk upgrade --no-cache \
-  && apk add --no-cache git bash ca-certificates
+FROM alpine:latest
```

Comment:
Any particular reason for this change?

Reply:
More of an unreflected copy & paste thing. I've reverted this back to the more sensible version pin :)